app/src/main/java/com/example/tasky/TaskyNavigation.kt

```diff
@@ -112,20 +115,79 @@ fun TaskyNavigation(
                     type = NavType.StringType
                 }
             )
-        ) {
+        ) { entry ->
             val viewModel: EventDetailViewModel = hiltViewModel()
 
+            viewModel.setEditedText(entry.savedStateHandle)
+
             with(viewModel.state) {
                 ObserveBoolean(navigateBack) {
                     navController.popBackStack()
                     viewModel.onEvent(EventDetailEvent.CloseClickResolved)
                 }
+
+                Observe(observer = navigateEditTitle) {
+                    navController.navigate(
+                        TaskyRoutes.EditScreen.getDestination(
+                            text = it,
+                            isTitle = true
```

Comment:
Again, a boolean here is a bit confusing

"isTitle" true

But if it´s not title, is it a description? A body? A footer?

Hard to understand for someone who doesn´t know about the code itself

Here you can again use a small Enum/Sealed class

Where you have TITLE and DESCRIPTION as options
And you just pass the enum/sealed class. Making it easier to read